api/redis/processReminders.js

```diff
@@ -0,0 +1,44 @@
+const reminderStore = require('./client.js').reminders;
+const bot = require('../bot.js');
+
+const getTimestamps = (array) => {
+  const timestamps = [];
+
+  array.forEach((timestamp) => {
```

Comment:
can use `return array.map` instead of `array.forEach`, get rid of the definition on ln 5, return on ln 12m and push on ln9

Reply:
This makes no sense. If the timestamps array definition is removed, where would the element be pushed on line 9?

Reply:
current:
```
const getTimestamps = (array) => {
  const timestamps = [];

  array.forEach((timestamp) => {
    if (timestamp < Date.now()) {
      timestamps.push(timestamp);
    }
  });
  return timestamps;
};
```

new (this will have null where the case is `>`):
```
const getTimestamps = (array) => {
  array.map((timestamp) => {
    if (timestamp < Date.now()) timestamp
  });
};
```

new (this will not have any nulls, exactly the same as original):
```
array.reduce((expired_timestamps, timestamp) => {
  if (timestamp < Date.now()) expired_timestamps.push(timestamp)
  return expired_timestamps;
}, []);
```